### PR TITLE
fs: implement transaction of fs creation

### PR DESCRIPTION
**Pull request by darkain84**
When error occured during fs creation
- rollback fs, policy
- rollback xfs quota

Change-Id: I7000bec1b68a1f8c204e83e9fb5642db155e17ee
Signed-off-by: HyunSeok Kil <hyunseok.kil@samsung.com>

### CODE REVIEW

File: src/server/fs/lib/fs-manager.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ var FsExtra = require('fs-extra');
 var express = require('express');
 var walkdir = require('walkdir');
 var async = require('async');
-var _ = require('underscore');
+var _ = require('lodash');
```
Comment (kyungmi) [inline src/server/fs/lib/fs-manager.js]: Why did you change library? Is there any side effect?

File: src/server/fs/lib/fs-manager.js [hunk 2]
```diff
@@ -1249,30 +1229,78 @@ router.post('/webida/api/fs',
...
                     }
-
-                    logger.info('addNewFS success', fsinfo);
-                    res.sendok(fsinfo);
                 });
-            });
+            },
+        ], function (err, result) {
+            if (err) {
+                return rollback(err, result);
```
Comment (kyungmi) [inline src/server/fs/lib/fs-manager.js]: I think that it is more safe to pass state value to a parameter of this rollback function.

File: src/server/fs/lib/fs-manager.js (hunk 1 above)
Comment (darkain84) [inline src/server/fs/lib/fs-manager.js]: Because I used the partialRight API which only supported on lodash module.
There was no side effect since lodash fully covered under score APIs
